src/uwtools/tests/drivers/test_forecast.py

```diff
@@ -1,26 +1,29 @@
-# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+# pylint: disable=missing-class-docstring, missing-function-docstring,protected-access,redefined-outer-name
```

Comment:
Because this file was reorganized into classes, it will be easier to see important diffs with the "hide whitespace" setting enabled (under the gear dropdown at the top of the PR "Files changed" tab)